apps/storybook/components/funke/BiometricsModal.stories.tsx

```diff
@@ -0,0 +1,87 @@
+        fill={theme['primary-500'].val}
+      />
+    </Svg>
+  )
+}
+
+const BiometricsModal = ({ onAcceptBiometrics, open, onOpenChange }: BiometricsModalProps) => {
+  return (
+    <Sheet
+      open={open}
+      onOpenChange={onOpenChange}
+      // FIXME: hardcoded for good fit in storybook, but this probably won't work on mobile...
+      snapPoints={[520]}
+      snapPointsMode="constant"
+      disableDrag
+      dismissOnOverlayPress={false}
+    >
+      <Sheet.Overlay
+        animation="lazy"
+        enterStyle={{ opacity: 0 }}
+        exitStyle={{ opacity: 0 }}
+        backgroundColor="$black"
+        opacity={0.2}
+      />
```

Comment:
Nit but this is my preset for the tamagui sheet. I find the default a bit too dark and slow.

```
<Sheet.Overlay
          bc="#00000033"
          animation="quick"
          enterStyle={{ opacity: 0 }}
          exitStyle={{ opacity: 0 }}
        />
```